diag_manager/diag_manager.F90

```diff
@@ -1607,7 +1607,7 @@ END FUNCTION send_data_3d
   LOGICAL FUNCTION diag_send_data(diag_field_id, field, time, is_in, js_in, ks_in, &
              & mask, rmask, ie_in, je_in, ke_in, weight, err_msg)
     INTEGER, INTENT(in) :: diag_field_id
-    CLASS(*), DIMENSION(:,:,:), INTENT(in), TARGET, CONTIGUOUS :: field
+    CLASS(*), DIMENSION(:,:,:), INTENT(in),TARGET,CONTIGUOUS :: field
```

Comment:
These changes are so weird !!!

Reply:
This is a result of the merge not going through correctly, and me having to re-add `target` and `contiguous` but not putting the spaces in when I did it.